source/exercises/index.rst

```diff
@@ -51,7 +51,7 @@ Lesson 4: Dictionaries, Sets, and Files
     file_lab.rst
     kata_fourteen.rst
 
-* :ref:`exercise_mailroom_plus`
+* :ref:`exercise_mailroom_part2_dict_files`
```

Comment:
Now that they are in their own file, they can be direct-linked into the index like the other files.

Reply:
got it.